funnel/views/index.py

```diff
@@ -94,6 +94,9 @@ def home(self) -> ReturnRenderWith:
             .order_by(Project.next_session_at.asc())
             .all()
         )
+        featured_profiles = (
+            Profile.query.filter_by(name=p).first() for p in current_app.config['FEATURED_PROFILES']
+        )
```

Comment:
Faster query:

`featured_profiles = Profile.query.filter(Profile.name_in(app.config.get('FEATURED_ACCOUNTS', []))).all()`

Note the config name change — we're deprecating the term Profile, so this is a bit in advance of the main PR. Also, read from `app` not `current_app`, as this view is limited to one app.